daemon/main.cpp

```diff
@@ -31,6 +31,13 @@
 #include "rtsp_server.hpp"
 #include "session_manager.hpp"
 
+#ifdef _USE_SYSTEMD_
+#include <systemd/sd-daemon.h>
+
+// with which interval we should pet the dog
+uint64_t current_watchdog_usec;
```

Comment:
move this declaration into main() function.

Reply:
Was mainly lazy, and re-used the ifdef. I'll move it.

Reply:
Now it's fixed.